sound/soc/sof/ipc.c

```diff
@@ -851,6 +852,26 @@ int snd_sof_ipc_valid(struct snd_sof_dev *sdev)
 }
 EXPORT_SYMBOL(snd_sof_ipc_valid);
 
+int sof_ipc_init_msg_memory(struct snd_sof_dev *sdev)
+{
+	struct snd_sof_ipc_msg *msg;
+
+	msg = &sdev->ipc->msg;
+	if (!msg->msg_data) {
+		msg->msg_data = devm_kzalloc(sdev->dev, sdev->host_box.size, GFP_KERNEL);
+		if (!msg->msg_data)
+			return -ENOMEM;
+	}
+
+	if (!msg->reply_data) {
+		msg->reply_data = devm_kzalloc(sdev->dev, sdev->host_box.size, GFP_KERNEL);
```

Comment:
they may be the same but just for sanity, shouldnt this be dsp_box.size?

Reply:
The reply data is copied from host_box

Reply:
The reply data is copied from host_box